nva-datamodel-java/src/main/java/no/unit/nva/api/PublicationResponse.java

```diff
@@ -155,49 +156,49 @@ public void setPublisher(Organization publisher) {
     }
 
     @Override
-    public List<ResearchProject> getProjects() {
-        return projects;
+    public URI getDoi() {
+        return doi;
     }
 
     @Override
-    public void setProjects(List<ResearchProject> projects) {
-        this.projects = projects;
+    public void setDoi(URI doi) {
+        this.doi = doi;
     }
 
     @Override
-    public EntityDescription getEntityDescription() {
-        return entityDescription;
+    public DoiRequest getDoiRequest() {
+        return doiRequest;
     }
 
     @Override
-    public void setEntityDescription(EntityDescription entityDescription) {
-        this.entityDescription = entityDescription;
+    public void setDoiRequest(DoiRequest doiRequest) {
+        this.doiRequest = doiRequest;
+        this.doiRequested = Objects.nonNull(doiRequest);
     }
 
     @Override
-    public URI getDoi() {
-        return doi;
+    public EntityDescription getEntityDescription() {
+        return entityDescription;
     }
 
     @Override
-    public void setDoi(URI doi) {
-        this.doi = doi;
+    public void setEntityDescription(EntityDescription entityDescription) {
+        this.entityDescription = entityDescription;
     }
 
     @Override
-    public DoiRequest getDoiRequest() {
-        return doiRequest;
+    public List<ResearchProject> getProjects() {
+        return Objects.nonNull(projects) ? projects : Collections.emptyList();
     }
 
     @Override
-    public void setDoiRequest(DoiRequest doiRequest) {
-        this.doiRequest = doiRequest;
-        this.doiRequested = Objects.nonNull(doiRequest);
+    public void setProjects(List<ResearchProject> projects) {
+        this.projects = projects;
     }
 
     @Override
     public List<URI> getSubjects() {
-        return subjects;
+        return Objects.nonNull(subjects) ? subjects : Collections.emptyList();
```

Comment:
Could also consider using Optional with .else(Collections.emptyList()) but this is fine.